packages/atlas/src/views/studio/YppDashboard/YppDashboard.tsx

```diff
@@ -46,6 +74,213 @@ export const YppDashboard: FC = () => {
+              title="Have more than one YouTube channel?"
+              description={
+                'You can apply to the YouTube Partner Program with as many YouTube & Atlas channels as you want. Each YouTube channel can be assigned to only one Atlas channel. \nYou can create a new channel from the top right menu.'
+              }
+            />
+          </>
+        )
+      case 'Settings':
+        return (
+          <>
+            <SettingsInputsWrapper>
+              <FormField
+                label="YouTube Sync"
+                description={
+                  <>
+                    {`With YouTube Sync enabled, ${atlasConfig.general.appName} will import videos from your YouTube channel over to Joystream. `}
+                    <Button _textOnly iconPlacement="right" icon={<StyledSvgActionArrowRight />}>
+                      Learn more
+                    </Button>
+                  </>
+                }
+              >
+                <OptionCardGroupRadio
+                  options={[
+                    { value: true, label: 'Sync YouTube videos', caption: 'Imports past and future videos' },
+                    { value: false, label: "Don't sync YouTube videos", caption: 'Pauses importing of future videos' },
+                  ]}
+                  selectedValue={isSync}
+                  onChange={setIsSync as (value: string | number | boolean) => void}
+                  direction={!mdMatch ? 'vertical' : 'horizontal'}
+                />
+              </FormField>
+              {isSync && (
+                <FormField
+                  label="Category of imported videos"
+                  description="Choose a category to be assigned to the imported videos by default. You can change it for each video later once it’s imported."
+                >
+                  <Select
+                    //todo: uncomment when isSync is supported by backend
+                    // disabled={currentChannel?.isSyncActive}
+                    items={categoriesSelectItems}
+                    onChange={setCategory}
+                    value={category}
+                  />
+                </FormField>
+              )}
+
+              <Divider />
+
+              <FormField
+                label="Danger zone"
+                description="By leaving the program you will no longer receive rewards for performing the tasks, and your future YouTube videos will not be imported automatically to Joystream. You will be able to connect your YouTube channel with another Joystream channel."
+              >
+                <Button
+                  variant="destructive-secondary"
+                  fullWidth
+                  size="large"
+                  onClick={() => openModal(openModalOptions)}
+                >
+                  Leave the program
+                </Button>
+              </FormField>
+            </SettingsInputsWrapper>
+            {areSettingsChanged && (
```

Comment:
I think we should have an animation here. That's what we have in other places where we use action bar